doc/0000-ccl_note/main.tex

```diff
@@ -787,69 +743,87 @@ \subsubsection*{Standard tracers}
-  \begin{equation}
-    b_{\rm IA} = - A_{\rm IA} \frac{C_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}
-  \end{equation}
-where $\rho_{\rm crit}$ is in the default \ccl units and $C_1= 5\times 10^{-14} (Mpc/h)^3 / (M_\odot/h)$ \citep{Brown02,Bridle07}. Note that although $\Omega_{\rm m}$ should in principle include non-relativistic neutrinos, this is not the case in \ccl at this point. 
+Here N.C. stands for ``number counts'', W.L. stands for ``weak lensing'' and
+CMB $\kappa$ is the CMB lensing convergence. $s(z)$ is the magnification bias,
+given as the logarithmic derivative of the number of sources with magnitude
+limit, and $r(\chi)$ is the angular comoving distance (see Eq. \ref{eq:angdist}).
+$f$ is the growth rate, which \ccl does not compute for massive neutrino cosmologies;
+therefore at this time an attempt to create a number count tracer in a cosmology
+with massive neutrinos will cause \ccl to raise an error. $C_\ell$ is instead
+computed assuming a linear-theory relation between the matter overdensity and
+peculiar velocity fields. While this should not be problematic for wide photometric
+redshift bins, users should exercise care when interpreting results for narrow
+window functions. For intrinsic alignments, (I.A. in the table), \ccl currently
+supports the so-called ``non-linear alignment model'', according to which the
+galaxy inertia tensor is proportional the local tidal tensor
+\cite{2004PhRvD..70f3526H,2007MNRAS.381.1197H}. $\chi_s$ is the distance to
+the source plane for CMB lensing. $b_{\rm IA}$ is the corresponding alignment
+bias. Notice that in the \ccl implementation, the normalization of this bias
+is chosen to coincide with the parameter $A_{\rm IA}$ used in current weak
+lensing analyses, i.e., the user is expected to pass $A_{\rm IA}$ to the routine,
+and this is connected to $b_{\rm IA}$ by the following expression\cite[Eq. 6]{Joachimi11}:
 
+\begin{equation}
+b_{\rm IA} = - A_{\rm IA} \frac{C_1 \Omega_{\rm m} \rho_{\rm crit}}{D(z)}
+\end{equation}
+where $\rho_{\rm crit}$ is in the default \ccl units and
+$C_1= 5\times 10^{-14} (Mpc/h)^3 / (M_\odot/h)$ \citep{Brown02,Bridle07}.
+Note that although $\Omega_{\rm m}$ should in principle include non-relativistic
+neutrinos, this is not the case in \ccl at this point.
 
-  Note that \ccl currently does not compute relativistic corrections to number counts \cite{2011PhRvD..84d3516C,2011PhRvD..84f3505B}. Although these should be included in the future, their contribution to the total fluctuation is largely subdominant (see \cite{GReffects} and the two references above), and therefore it is safe to work without them in most cases.
 
-  It is also worth noting that the equations above should be modified for non-flat cosmologies by replacing the spherical Bessel functions $j_\ell$ with their hyperspherical counterparts \cite{1994ApJ...432....7K}. This will be revisited in future versions of \ccl.
-  
-\subsubsection*{Limber integrals}
```

Comment:
If we are removing this, it might be worth referring the reader to the CCL paper for these expressions.

Reply:
This has not been removed.